release-calendar/src/client/components/EventCard.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/eventCard.scss';
+import * as React from 'react';
+import {EventApi} from '@fullcalendar/core';
+import moment from 'moment';
+
+export interface EventCardProps {
+  eventApi: EventApi;
+}
+
+export class EventCard extends React.Component<EventCardProps, {}> {
+  //TODO: use readable channel titles in eventCard
+
+  content = [
```

Comment:
Instead of entered and exit, what if it showed the whole history? Something like
```
Created as Nightly on xx
Promoted to Opt-In Experimental, Opt-In Beta on xx
Promoted to 1% Experimental, 1% Beta on xx
Promoted to Stable on xx
Promoted to LTS on xx
```